src/main/resources/templates/macros/report/elements.vm

```diff
@@ -41,6 +41,10 @@
                 </table>
               #end
 
+              #if ($step.getDocString())
+                <pre>$step.getDocString().getValue()</pre>
```

Comment:
Maybe you can simple reuse or copy this https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/main/resources/templates/macros/report/steps.vm#L26 so expandable will be quite simple to add